Godeps/Godeps.json

```diff
@@ -51,7 +50,7 @@
 		},
 		{
 			"ImportPath": "github.com/mattn/go-isatty",
-			"Rev": "fdbe02a1b44e75977b2690062b83cf507d70c013"
+			"Rev": "ae0b1f8f8004be68d791a576e3d8e7648ab41449"
```

Comment:
Could you please update to latest go-isatty (7fcbc72f853b92b5720db4a6b8482be612daef24)?